resources/assets/components/CampaignIdSingle/index.js

```diff
@@ -81,29 +82,19 @@ class CampaignIdSingle extends React.Component {
             <p>–</p>
           )}
           <h4>Start Date</h4>
-          <p>{new Date(campaign.start_date).toDateString()}</p>
-
+          <p>{format(new Date(campaign.start_date), 'MM/dd/yyyy')}</p>
```

Comment:
I just want to double-check that this is working in Safari now.

I haven't pulled down the branch to my local, but just testing this snippet on QA now and it seems like `new Date(window.STATE.campaign.start_date)` still results in `Invalid Date`.

Reply:
@DFurnes ah thanks for pointing out! When I hit rogue.test in Safari, it sends me to the google search for that. Is there anything I have to update to be able to hit local in Safari?

Reply:
ack I got this to work now - sorry for all the pinging!

Reply:
@DFurnes you're right, this doesn't seem to work either: 
![Screen Shot 2019-03-12 at 1 50 08 PM](https://user-images.githubusercontent.com/9019452/54223432-dc81f680-44cd-11e9-8440-af4acca13e5c.png)

Looks like because `start_date` and `end_date` are `DATETIME` in the database and probably should be a `TIMESTAMP`?

Reply:
@DFurnes updated with some regex and now it seems to be working both in Chrome and Safari!